lib/ansible/modules/packaging/language/pip.py

```diff
@@ -271,15 +286,20 @@ def _get_packages(module, pip, chdir):
     return (command, out, err)
 
 
-def _is_present(name, version, installed_pkgs, pkg_command):
+def _is_present(module, name, version, installed_pkgs, pkg_command):
     '''Return whether or not package is installed.'''
+    if version is not None and version != "":
+        try:
+            vp = VersionPredicate("%s (%s)" % (name, _version_number_to_formula(version)))
```

Comment:
Why use parens syntax?

Reply:
VersionPredicate doesn't support `Django >=1.11.1` but only support `Django (>=1.11.1)`. They need brackets to help them separate name and version.

Reply:
I've added a comment for you to consider switching to pkg_resources